<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Segmentation Comparer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="base">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;base</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="seg1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>seg&amp;1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="seg2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>seg&amp;2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dsc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;DSC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>View</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="axialButton">
           <property name="text">
            <string>Axial</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="coronalButton">
           <property name="text">
            <string>Cor.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="saggitalButton">
           <property name="text">
            <string>Sag.</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
        <property name="numDigits">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sliderLayout">
        <item>
         <widget class="QSlider" name="slice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="invertedAppearance">
           <bool>true</bool>
          </property>
          <property name="invertedControls">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="image">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
